src/User32.Desktop/User32.cs

```diff
@@ -647,13 +1103,71 @@ public static partial class User32
         [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Unicode)]
         public static extern int RegisterWindowMessage(string lpString);
 
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern bool GetWindowDisplayAffinity(IntPtr hWnd, out int dwAffinity);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern bool EnumDisplayMonitors(IntPtr hdc, IntPtr lprcClip, MonitorEnumDelegate lpfnEnum, IntPtr dwData);
+
+        [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Unicode)]
+        public static extern bool GetClassInfoEx(IntPtr hInstance, string lpClassName, ref WNDCLASSEX lpWndClass);
```

Comment:
And please type the parameters as `IntPtr, [Friendly(In|Array)] char*, [Friendly(Out)] WNDCLASSEX*`